__tests__/integration.test.js

```diff
@@ -60,4 +60,49 @@ describe("/api/topics", () => {
     })
   })
 
+});
+
+// api/articles/:article_id
+
+describe("/api/articles/:article_id", () => {
+
+  test("GET: Requests to this endpoint should respond with single object with an id matching the request and the following properties; author, title, article_id, body ,topic, created_at, votes, article_img_url", () => {
+    return request(app)
+    .get("/api/articles/3")
+    .expect(200)
+    .then(({body})=>{
+      const {article} = body
+      expect(article).toMatchObject({
+        article_id: expect.any(Number),
```

Comment:
We can be more specific than expect.any(Number) here, we know what article_id we should get back 👍